lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.scss

```diff
@@ -1,50 +1,35 @@
 adf-search-chip-autocomplete-input {
     .adf-chip-list {
         width: 100%;
-    }
-
-    .mat-form-field-appearance-outline .mat-form-field-outline {
-        top: 0;
-    }
 
-    .mat-form-field-infix {
-        border: none;
+        /* stylelint-disable-next-line */
```

Comment:
I would like to leave that one as this form-field requires many changes from default and many HTML elements requires changes, and it's not possible to get to some, without direct using angular material class name